frontend/src/test/scala/bloop/tasks/TestTaskTest.scala

```diff
@@ -30,45 +31,58 @@ object TestTaskTest extends DynTest {
     assert(withSuffix.get.name == "with-tests-test")
   }
 
-  import java.net.URLClassLoader
-  private val testLoader: ClassLoader = {
-    val classpath = Tasks.constructClasspath(testProject)
-    new URLClassLoader(classpath, TestInternals.filteredLoader)
+  private def processConfig(fork: Boolean): ProcessConfig = {
+    val javaEnv = JavaEnv.default(fork)
+    val classpath = testProject.classpath
+    ProcessConfig(javaEnv, classpath)
```

Comment:
Can we have a more descriptive way? Something along the lines of `ProcessRunnerConfig`?